templates/vue/src/components/modals/NodeModal/forms/ContentForm/ChoiceRow.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div v-if="multipleAnswerSelected" class="container">
+    <span id="handle" v-handle class="fas fa-bars fa-s"></span>
+    <b-form-checkbox :value="item.id" />
+    <b-form-group v-if="useImages">
+      <file-upload
+        v-model="item.imageurl"
+        input-test-id="node-choiceRow-thumbnail-url"
+        :data-qa="`choicerow-checkbox-thumbnail-${item.id}`"
+        :show-url-upload="false"
+        :show-image-preview="true"
+        :compact-mode="true"
+        file-types="image/*"
+        @isUploading="handleUploadChange"
+      />
+    </b-form-group>
+    <div :class="useImages ? 'inputContainerImage' : 'inputContainer'">
+      <b-form-input
+        v-model="item.value"
+        class="formInput"
+        :placeholder="placeholder"
+        :data-qa="`choicerow-checkbox-input-${item.id}`"
+      ></b-form-input>
+    </div>
+    <b-button
+      :disabled="removeButtonDisabled"
+      squared
+      variant="outline-danger"
+      @click="$emit('remove')"
+    >
+      Remove
+    </b-button>
+  </div>
+
+  <div v-else class="container">
+    <span id="handle" v-handle class="fas fa-bars fa-s"></span>
+    <b-form-checkbox
+      :value="item.id"
+      :disabled="isDisabled && selectedRadioChoice != item.id"
+    >
+      <b-form-group v-if="useImages">
+        <file-upload
+          v-model="item.imageurl"
+          input-test-id="node-choiceRow-thumbnail-url"
+          :data-qa="`choicerow-radio-thumbnail-${item.id}`"
+          :show-url-upload="false"
+          :show-image-preview="true"
+          file-types="image/*"
+          @isUploading="handleUploadChange"
+        />
+      </b-form-group>
+    </b-form-checkbox>
+    <div :class="useImages ? 'inputContainerImage' : 'inputContainer'">
+      <b-form-input
+        v-model="item.value"
+        class="formInput"
+        :placeholder="placeholder"
+        :data-qa="`choicerow-radio-input-${item.id}`"
+      ></b-form-input>
+    </div>
+    <b-button
+      :disabled="removeButtonDisabled"
+      squared
+      variant="outline-danger"
+      @click="$emit('remove')"
+    >
+      Remove
+    </b-button>
+  </div>
+</template>
+
+<script>
+import FileUpload from "@/components/modals/common/FileUpload"
+import { HandleDirective, ElementMixin } from "vue-slicksort"
+export default {
+  components: {
+    FileUpload,
+  },
+  directives: { handle: HandleDirective },
+  mixins: [ElementMixin],
+  props: {
+    question: {
+      type: Object,
+      required: true,
+    },
+    multipleAnswerSelected: {
+      type: Boolean,
+      required: true,
+    },
+    index: {
+      type: Number,
+      required: true,
+    },
```

Comment:
@wynnset, after removing this prop, it seems that vue will complain if there is no index prop passed for a v-for element.
<img width="490" alt="Screen Shot 2021-07-05 at 6 01 27 PM" src="https://user-images.githubusercontent.com/30706624/124527731-24be3480-ddbb-11eb-9c41-95a03e8149cf.png">

Reply:
Yeah, you need to keep the index prop on your element because you are using the v-for, but you do not need to include it in your prop list in your actual component.